setup.py

```diff
@@ -11,16 +11,14 @@
     author_email='aroy@alum.mit.edu',
     url='https://github.com/audreyr/cookiecutter-pypackage',
     keywords=['cookiecutter', 'template', 'package', ],
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+    python_requires='>=3.5, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

Comment:
Left `!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',` in to demonstrate how that's done though it's redundant.